comp/core/tagger/taggerimpl/tagger.go

```diff
@@ -46,6 +48,21 @@ import (
 
 var entityIDRegex = regexp.MustCompile(`^en-(init\.)?([a-fA-F0-9-]+)/([a-zA-Z0-9-_]+)$`)
 
+const (
+	// External Data Prefixes
+	// These prefixes are used to build the External Data Environment Variable.
+	// This variable is then used for Origin Detection.
+	externalDataInitPrefix          = "it-"
+	externalDataContainerNamePrefix = "cn-"
+	externalDataPodUIDPrefix        = "pu-"
```

Comment:
open question: Does it makes sense to have it publicly defined to be reuse by third party project like the DSD/APM libraries.

Reply:
Checked together, we will see how we do the implementation for Traces and improve this then if needed.